src/js/main.js

```diff
@@ -419,8 +404,8 @@ function startProcess() {
 
     // listen to all input change events and adjust the value within limits if necessary
     $("#content").on('focus', 'input[type="number"]', function () {
-        var element = $(this),
-            val = element.val();
+        const element = $(this),
+              val = element.val();
```

Comment:
is this making val a const?

Reply:
In general I am not a fan of multiple declarations in the same statement, I'd rather do it this way:

```
const element = $(this);
const val = element.val();
```

This makes it obvious that `val` being a `const` is intended and not just an oversight.